src/models/base_types.py

```
@@ -0,0 +1,101 @@
+from enum import Enum
+from typing import Protocol, Dict, Any, Optional, List
```

Ruff (0.8.2)

`typing.Dict` imported but unused

Remove unused import: `typing.Dict`

(F401)